Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN \
   cd /app && \
   npm ci
 
-EXPOSE 8000
+EXPOSE ${PORT:-8080}
```

Comment:
You can remove this line. The port variable won't exist at build time, and EXPOSE doesn't do much anyways